versioned_docs/version-3.0.0/folders.md

```diff
@@ -1,76 +1,76 @@
+
+#### `config/`
+
+The `config` directory contains the config files needed for the electron app.  This includes the event-marker details and event codes.
+
+In the `config/` directory, there are `.js` files which contain settings for the different parts of the task.  Every task should have a `main` config and a `trigger` config (assuming use of the event marker). The `main` config has all global settings for the task (such as whether it is in mturk mode or not), load the appropriate language file, and set up a default (or only) configuration object for the task. Different in-task features can be loaded from .env files and is set in the `main` config. For example, we have included some of these files in the `env` directory (e.g. `env.clinic` file enables event marker, photodiode and volume adjustment). The `trigger` config has settings specific to the event marker and uses a slightly different style of javascript as it is imported both in the React app as well as the electron process.
+
+Other config files can be used to add settings for specific blocks or sub-sections of the experiment. Also, *note that the productId can be overwritten by the environment variable EVENT_MARKER_PRODUCT_ID*.
+
+### `src/`
+
+This folder contains the code for the app, the vast majority of changes and code should go here.
+
+#### `App.jsx`
+
+This is the starting point for the app. The `<Login>` component handles user authentication in the different use cases and the `<JsPsychExperiment>` component initializes the `jspsych` 7 experiment. This is also where communication is set up with the `electron` and `psiturk` processes.
+
+#### `App.css`
+
+This is where styling for the app is housed. If colors, fonts, spacing, etc. need to be set, do it here.
+
+#### `components/`
+
+This folder contains the components referenced in `App.jsx`.  This includes the Honeycomb `<JsPsychExperiment>` component which connects the Honyecomb login page to your `jspsych` 7 experiment.
+
+#### `language/`
+
+Any language that is displayed in the experiment should be stored in this folder. Usage of language json files allows for easy internationalization of the task (e.g. english and spanish) as well as allows for mturk specific language. This also makes it easy to re-use common phrases in many places in the task.
+
+#### `lib/`
+
+The `lib/` directory contains utility functions and markup that is used in the tasks.  This allows for functions and html to be re-used wherever needed. The `lib/utils.js` file contains functions that are generally useful across many tasks, whereas `lib/taskUtils.js` contains functions specific to this task.
+
+#### `lib/markup`
+
+`markup` files should contain primarily templates for HTML that is used throughout the task. Typically this will be a function that takes in some parameters and then returns a string with html.

```

Comment:
```suggestion
`markup` files should contain primarily templates for HTML that are used throughout the task. Typically this will be a function that takes in some parameters and then returns a string with html.

```